cmd/mdatagen/internal/templates/telemetrytest_test.go.tmpl

```diff
@@ -3,13 +3,15 @@
 package {{ .Package }}test
 
 import (
-    "context"
+	"context"
 	"testing"
 
 	"github.com/stretchr/testify/require"
+	"go.opentelemetry.io/otel/metric"
 	"go.opentelemetry.io/otel/sdk/metric/metricdata"
 	"go.opentelemetry.io/otel/sdk/metric/metricdata/metricdatatest"
 
+	"{{ getModule }}/internal/metadata"
```

Comment:
This import was not originally present, and the import error would have been resolved by running `make fmt`. This can be a bit confusing for newcomers like myself, as I was unaware that `make fmt` would automatically fix such issues. Hence, I think it makes sense to add the import in `generate`.

The `getModule` function retrieves the module name using `ScopeName`. Although this field is customizable, I read in the documentation that it is recommended to leave it empty. Therefore, I believe it is safe to use in this context. Please let me know if this approach is unsuitable.

Reply:
Let's split this in a separate change since it is a bit controversial. We can move forward immediately with everything else.

Reply:
removed this in the latest commit.

Reply:
I'm not sure why this change would be controversial? I guess I'm missing some context here.

telemetry based generated code will fails expecting `metadata.NewTelemetryBuilder` to be defined as an import in the generated tests, with gofmt tools adding necessary imports